OMEdit/OMEditLIB/FlatModelica/Expression.cpp

```diff
@@ -1342,7 +1344,7 @@ namespace FlatModelica
 
   void Enum::print(std::ostream &os) const
   {
-    os << _name;
+    os << _name << '(' << _index << ')';
```

Comment:
I'm not sure this is a good idea since the intent is to just print the Modelica representation of the expression, which for an enum literal is just its name.